zulip_bots/zulip_bots/bots/connect_four/doc.md

```diff
@@ -0,0 +1,95 @@
+# Connect Four Bot
+
+The Connect Four bot is a Zulip bot that will allows users
+to play a game of Connect Four against either another user,
+or the computer. All games are run within private messages
+sent between the user(s) and the bot.
+
+Starting a new game with another user requires a simple command,
+and the desired opponents zulip-related email adress:
+
+```
+@<bot-name> start game with user@example.com
+```
+
+Starting a game with the computer is even simpler:
+
+```
+@<bot-name> start game with computer
+```
+
+**See Usage for a complete list of commands**
+
+*Due to design contraints, the Connect Four Bot
+can only run a single game at a time*
+
+## Setup
+
+The Connect Four Bot does not require a config file or API key.
+It can be used without setup
+
+## Usage
+
+*All commands should be prefaced with* ```@<bot-name>```
+
+1. ```help``` : provides the user with relevant
+commands for first time users.
+
+<br>
+
+2. ```status``` : due to design contraints, the
+bot can only run a single game at a time. This command allows
+the user to see the current status of the bot, including
+whether or not the bot is running a game, if the bot is waiting
+for a player to accept an invitation to play, as well as who
+is currently using the bot.
+
+<br>
```

Comment:
I would suggest removing all of the `<br>` tags. You shouldn't need these in Markdown! :)